packages/nimble-components/src/table-column/text/tests/table-column-text.spec.ts

```diff
@@ -201,4 +201,53 @@ describe('TableColumnText', () => {
             );
         }
     });
+
```

Comment:
Could we add some cases to ensure the tooltip for truncated text is working? You should be able to copy logic from elsewhere in this file where we verify that for the cell itself.

Reply:
Done! Even seemed to find a valid issue to fix.